drkafka/config/doctorkafka.properties

```diff
@@ -21,6 +21,25 @@
 # [required] zookeeper quorum for storing doctorkafka metadata
 doctorkafka.zkurl=zookeeper001:2181,zookeeper002:2181,zookeeper003:2181
 
+# [required] doctorkafka web port
+doctorkafka.web.port=8080
+
+# [optional] disable doctorkafka service restart
+doctorkafka.restart.disabled=false
+
+# [required] doctorkafka service restart interval
+doctorkafka.restart.interval.seconds=86400
+
+# [optional] ostrich port #
+doctorkafka.ostrich.port=2052
+
+# [optional] tsd host and port.
+doctorkafka.tsd.hostport=localhost:18621
+
+# [required] legacy config for UI to read from the report topic, will be removed in the future
+# should be set same as doctorkafka.actions.report_operation.config.topic
+doctorkafka.action.report.topic=operator_report
```

Comment:
will there be a setting on  action.report.zkurl?  if there is one, shall we group all related configuration together? 

are we planning to allow users to write plug-ins on action report?  wil that be over-design?